src/components/molecules/tabbed-navigation/tabbed-navigation.module.scss

```diff
@@ -2,6 +2,10 @@
 @use "../../../sass/settings";
 @use "../../../sass/typography";
 
+.tabbed-navigation {
+  display: block;
+}
+
```

Comment:
is this just here so that it shows the class in the dom?

Reply:
Yes. I think we need to have a nav tag around the tabbed navigation